store/storetest/link_metadata_store.go

```diff
@@ -116,7 +116,7 @@ func testLinkMetadataStoreSave(t *testing.T, ss store.Store) {
 		// Should return the original result, not the duplicate one
 		linkMetadata, err = ss.LinkMetadata().Get(metadata.URL, metadata.Timestamp)
 		require.NoError(t, err)
-		assert.Equal(t, &model.PostImage{}, linkMetadata.Data)
+		assert.NotEqual(t, &model.PostImage{}, linkMetadata.Data)
```

Comment:
A stronger check would be to check against the original data and verify that they are same.